cli/requirements.txt

```diff
@@ -3,6 +3,6 @@ tox>=2.2, <3.0
 wheel>=0.24.0, <1.0
 pytest>=2.9.1
 teamcity-messages>=1.20
-PyInstaller==3.2.1
+PyInstaller==3.3
```

Comment:
I had to bump this as otherwise binaries couldn't be generated with Python 3.6 (failing with an error).